client/src/state/editor/readBlocksForAreaQuery.js

```diff
@@ -5,25 +5,22 @@ import gql from 'graphql-tag';
 // ElementalAreaIfExists. The results of the query must be set to the "blocks" prop on
 // the component that this HOC is applied to for binding implementation.
 const query = gql`
-query ReadBlocksForPage($id:ID!) {
-  readOnePage(ID: $id, Versioning: {
+query ReadBlocksForArea($id:ID!) {
+  readOneElementalArea(ID: $id, Versioning: {
     Mode: DRAFT
   }){
-    ID
-    ElementalAreaIfExists {
-      Elements {
-        pageInfo {
-          totalCount
-        }
-        edges {
-          node {
-            ID
-            Title
-            BlockSchema
-            IsLiveVersion
-            IsPublished
-            Version
-          }
+    Elements {
```

Comment:
Does this mean we can remove the extension that provides `ElementalAreaIfExists` now?

Reply:
Ah yep I forgot to do that.

Reply:
Ok I've removed that extension as part of this PR 🙂